docs/content/docs/connectors/table/filesystem.md

```diff
@@ -225,6 +225,12 @@ To define when to commit a partition, providing partition commit trigger:
         <td>Duration</td>
         <td>The partition will not commit until the delay time. If it is a daily partition, should be '1 d', if it is a hourly partition, should be '1 h'.</td>
     </tr>
+    <tr>
+        <td><h5>sink.partition-commit.watermark-time-zone</h5></td>
```

Comment:
The `partition-time` is a string with TIMESTAMP meaning, and I think it's not correct to say the `partition-time` has time zone. And the watermark is a long value, it can be explained as UTC timestamp mills or Instant mills, I tend to  use current option name